extensions/amp-analytics/0.1/vendors/navegg.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * Copyright 2018 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export const NAVEGG_CONFIG = /** @type {!JsonObject} */ ({
+  'requests': {
+    'host': 'amp.navdmp.com',
+    'path': 'amp',
+    'params':
```

Comment:
Same `params` is not `requests`, you can include it as `extraUrlParams`, or embed it to the `pageview` request. 

https://www.ampproject.org/docs/reference/components/amp-analytics#extra-url-params

Reply:
Added to pageview.